File: DayFive/Java/AfterBreak/Tasks.java
public class Tasks{


public static int addDigits(int number){
    String newNumber = String.valueOf(number);
    String [] array = newNumber.split("");
    int result = 0;

    while(array.length != 1){
        for(String each : array){
            result += Integer.parseInt(each);
        }
         String newresult = String.valueOf(result);

         if(newresult.length() > 1){
            array = newresult.split("");
            result = 0;
        }
         else{ break;}
    }
    return result;

}
    


public static int getSingleDigit(int [] array){
    int count = 0;
    int single = 0;
    String multiple = "";

    for(int element : array){
        for(int ele : array){
            if(element == ele){
                count ++;
                single = element;
            }
        }

        if(count == 1){
            multiple  += single;
        }
       count = 0;
       single = 0;
    }
    single = Integer.parseInt(multiple);
    return single;

}



public static void main(String...ugo){
    int[]array = {1,2,2,3,3,4};
    System.out.println(getSingleDigit(array));

}

}

File: DayFive/JavaScript/AfterBreak/Tasks.js

let addDigits = (number) => {
    let newNumber = String(number);
    let result = 0;

    while(newNumber.length > 1){
        for(let each of newNumber){
            result += Number(each);
        }
         let newresult = String(result);
         if(newresult.length > 1){
            newNumber = newresult;
            result = 0

         }
         else{ break}
    }
    return result;

}


    


let getSingleDigit = (array)=>{
    let count = 0;
    let single = 0;
    multiple = "";

    for(let element of  array){
        for(let ele of array){
            if(element == ele){
                count +=1;
                single = element;
            }
        }

        if(count == 1){
            multiple  += single;
        }
       count = 0;
       single = 0;
    }
    single = Number(multiple);
    return single;

}

console.log(addDigits(38))
array = [1,2,2,6]
console.log(getSingleDigit(array))







console.log(get_single_digit_in_array(38))
